### ISSUE TITLE
allow widget in  CupertinoActionSheet title receive click events.

### ISSUE THREAD

**Issue by fullflash** [flutter/flutter#154011]
### Use case

currently the widget inside title property are not receiving tap event could not found a a blocker inside source code.

### Proposal

having clickableTitle property for CupertinoActionSheet to allow title widget receive tap event.

**Comment by SuicaLondon** [comment]
From the [document](https://developer.apple.com/design/human-interface-guidelines/action-sheets), I believe that was intentionally designed. We can check the title and the description of the action sheet on watchOS, the title and description is just a description without any functionality.

![](https://docs-assets.developer.apple.com/published/e0d8c87ceb6d9d6b67491d212d276920/action-sheet-series-8@2x.png)

Flutter did implement and following the design of the iOS for the consistency. 

```Dart
/// This action sheet styles its title and message to match standard iOS action
/// sheet title and message text style.
```

Back to your requirements, I suggest: 
1. Does not change anything in the source code of `CupertinoActionSheet`. Add the `GestureDetector` to the `title` widget
2. Does not change anything in the source code of `CupertinoActionSheet`. Implement a new action sheet instead.

**Comment by huycozy** [comment]
> Add the GestureDetector to the title widget

@fullflash Doesn't this work for your case? If so, can you share a minimal sample code that produce the issue?
